src/shared/components/data-table/components/table-controls/components/table-actions/components/download-data/DownloadData.tsx

```diff
@@ -137,7 +137,7 @@ const DownloadData = () => {
   );
 };
 
-const formatCSV = (table: (string | number)[][]) => {
+const formatTSV = (table: (string | number)[][]) => {
   return table.map((row) => row.join(',')).join('\n');
```

Comment:
see the separator

Reply:
After all the effort of renaming, it still gives back a CSV ;-) 
```suggestion
  return table.map((row) => row.join('\t')).join('\n');
```

Reply:
Ah I missed this one. I made the same change here as well:
https://github.com/Ensembl/ensembl-client/pull/941/files#diff-e49005527206e2193fbbe69d64f07708ab87d2f9323a8596cbe1550b166e4867R193

Anyway why are we doing the same thing (function) in two separate files? Can't we do it in just one place?